cookbook/import_export/create-connector.rst

```diff
@@ -164,6 +165,87 @@ Register it in the form provider and you are good to go!
     :linenos:
     :lines: 8-13
 
+Configure a custom field in our UI for Job
+------------------------------------------
+
+If you want to add a custom field such as a "select" field with custom data, you will have to create your own JS module with dynamic choices.
+
+First, you will have to create a controller (with a conform routing configuration) which return the list on a JSON format for example:
+
+.. code-block:: php
+    
+    class MyCustomController
+    {
+        ...
+
+        public function listAction() {
+            $choices['my_option_1'] = 'my_option_1';
+            $choices['my_option_2'] = 'my_option_2';
+
+            return new JsonResponse($choices);
+        }
+    }
+
+Then configure a fetcher
+
+.. literalinclude:: ../../src/Acme/Bundle/DummyConnectorBundle/Resources/config/requirejs.yml
```

Comment:
Yup, the Python validator for YAML files is "stricter" than Symfony 2 one: note that in Symfony 3 this syntax is also invalid:

```yaml
foo: %foo% # valid in Symfony2, invalid in Symfony3 and in YAML reference
foo: "%foo%" # valid everywhere
```